src/interface/mesh_data.hpp

```diff
@@ -477,6 +479,14 @@ class MeshData {
 
   std::vector<bool> send_buf_alloc_status_, set_buf_alloc_status_,
       restrict_buf_alloc_status_;
+ 
+ public: 
+  
+  cell_centered_bvars::BufferCache_t send_bnd_info{}; 
+  cell_centered_bvars::BufferCache_t::host_mirror_type send_bnd_info_h{}; 
+  
+  cell_centered_bvars::BufferCache_t recv_bnd_info{}; 
+  cell_centered_bvars::BufferCache_t::host_mirror_type recv_bnd_info_h{}; 
```

Comment:
Wait... wasn't the boundary info already in `mesh_data`? Or did I miss something?

Reply:
Yes, there are already `BufferCache_t` objects in `mesh_data`. I just added another set so that this was completely divorced from the old machinery.